docs/docs/plugins-themes-and-starters.md

```diff
@@ -36,3 +36,72 @@ Gatsby offers [official starters](/docs/starters/#official-starters) for a defau
+
+#### Versioning
+
+Starters can still be versioned inside a repository so that you could track issues or bugs associated with specific developments in the starter but it's more difficult to make updates with new changes.
+
+#### Installing as a package
+
+Because starters are "ejected" each time they are used, it's difficult to sync changes across multiple sites derived from the same starter. This problem was one of the motivating factors in developing the newer concept of themes for Gatsby. You can read more about why themes were developed in the [What are Gatsby Themes guide](/docs/themes/what-are-gatsby-themes/#gatsby-starters).
+
+### Differences in configurability
+
+Plugins and themes can expose a varying level of an API surface (meaning a piece of the plugin or theme that a user of it can interact with), constraining what is possible and providing sensible defaults to make them easier to start picking up and using.
+
+| Configuration         | Plugin | Theme | Starter |
+| --------------------- | ------ | ----- | ------- |
+| Shadowing             | ◐      | ●     | ○       |
+| Uses Multiple Plugins | ◐      | ●     | ●       |
+| Custom components     | ◐      | ●     | ●       |
+
+#### Shadowing
+
+The options exposed by a plugin or theme are up to the author. For example, a plugin or theme can allow a specific path to be provided in the `gatsby-config` so the plugin knows where to build pages from. In this case the user wouldn't be able adjust _how_ those pages are built, only from what path. As an escape hatch to change small pieces of an installed theme, [shadowing](/docs/themes/shadowing/) was introduced. Theme shadowing allows users to replace a file with their own version of it. Because themes are also plugins, shadowing is possible in plugins as well.
+
+An example of a plugin that uses shadowing is [`gatsby-plugin-theme-ui`](/packages/gatsby-plugin-theme-ui/?=theme-ui#customizing-the-theme) which allows you to shadow a theme file to use your own theme.
+
+Starters aren't capable of shadowing (and they don't need to be), because a user of a starter can adjust any file by editing it directly.
+
+#### Uses multiple plugins
+
+Starters can be configured with multiple plugins so someone can get up and running without worrying about configuring too many loose ends. Themes are capable of the same by making a `gatsby-config` that a Gatsby site will run along with its own config. Plugins are also capable of this (because themes are a plugin), but it is recommended that _plugins be limited to their specific functionality_.
+
+#### Custom components
+
+Custom components are most traditionally distributed as packages in the React ecosystem. Components don't need to hook into the Gatsby build system, so if shipped with a plugin don't need to be included in a `gatsby-config`'s plugin array. This is the case with `gatsby-image` which is a React component. It doesn't need to be included in the plugins array because it is merely a component.
```

Comment:
```suggestion
Custom components are most traditionally distributed as packages in the React ecosystem. Components don't need to hook into the Gatsby build system, so if shipped with a plugin they don't need to be included in a `gatsby-config`'s plugin array. This is the case with `gatsby-image` which is a React component. It doesn't need to be included in the plugins array because it is merely a component.
```